packages/reactive-element/src/css-tag.ts

```diff
@@ -172,9 +172,12 @@ export const adoptStyles = (
   styles: Array<CSSResultOrNative>
 ) => {
   if (supportsAdoptingStyleSheets) {
-    (renderRoot as ShadowRoot).adoptedStyleSheets = styles.map((s) =>
-      s instanceof CSSStyleSheet ? s : s.styleSheet!
-    );
+    const sheets = styles.map(
+      (result) => (result as CSSResult).styleSheet ?? result
+    ) as CSSStyleSheet[];
+    const defaultView = renderRoot.ownerDocument.defaultView ?? window;
+    const viewSheets = sheetsForView(defaultView, sheets);
```

Comment:
Let's not do this if `defaultView === window`?

Reply:
Ok, I think I get where you're coming from. My only issue is that `window` may not be the origin of instantiation of the element.

Reply:
The coalescing was not present initially. I only added it because typescript kept yelling at me for the possibility of `defaultView` being null.